pom.xml

```diff
@@ -174,12 +174,12 @@
             <dependency>
                 <groupId>jakarta.xml.bind</groupId>
                 <artifactId>jakarta.xml.bind-api</artifactId>
-                <version>${dependency.jakarta-ee-jaxb-api.version}</version>
+                <version>4.0.0</version>
             </dependency>
             <dependency>
                 <groupId>org.glassfish.jaxb</groupId>
                 <artifactId>jaxb-runtime</artifactId>
-                <version>${dependency.jakarta-ee-jaxb-api.version}</version>
+                <version>4.0.3</version>
```

Comment:
If we manage the `jaxb-runtime` version we should also manage the `jaxb-core` version